services/admin/logging/internal/wssd.go

```diff
@@ -66,8 +67,34 @@ func (c *client) GetLogFile(ctx context.Context, location, filename string) erro
 	return nil
 }
 
+func (c *client) SetLogLevel(ctx context.Context, setloglevel uint32) error {
+	request := setLogLevelRequest(setloglevel)
+	fileStreamClient, err := c.LogAgentClient.Set(ctx, request)
+	if err != nil {
+		return err
+	}
+	getLogFileResponse, innerErr := fileStreamClient.Recv()
+
+	if innerErr != nil {
+		return innerErr
+	}
+
+	if getLogFileResponse.Done {
+		return nil
+	}
+
+	Err := errors.New("error setting tracelevel")
+	return Err
+}
+
 func getLoggingRequest(location string) *wssdadmin.LogRequest {
 	return &wssdadmin.LogRequest{
 		Location: location,
 	}
 }
+
+func setLogLevelRequest(setloglevel uint32) *wssdadmin.SetRequest {
+	return &wssdadmin.SetRequest{
+		Settracelevel: setloglevel,
```

Comment:
nit: let's keep the naming consistent. Either use trace level or log level. Not both.